src/collections/posts/where-are-my-bookmarks.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Where Are My Bookmarks?
+section: articles
+category: guide
+publish: 2024-06-15T11:42:00.000Z
+homepageText: With the re-write of the BeastSaber site complete, bookmarks at
+  BSaber have become a thing of the past. Learn how to recover your legacy
+  bookmarks and how to import them to BeatSaver!
+image: /static/uploads/posts/fabookmark.png
+authors:
+  - '4285984'
+showInPostListing: true
+---
+
+As you've surely noticed, the BeastSaber site has undergone a huge change. While we know that change can bring a lot of mixed emotions and feelings, the changes here were needed as the old BeastSaber site was slowly but surely falling apart and would've likely become completely unusable.
+
+Fortunately, thanks to the tremendous work by our Developers, we tried our best to save the some of the most important things that would have otherwise been lost forever, including bookmarks!
+
+# Where Do I Find My Old Bookmarks?
+
+Thanks to [TopCat](https://beatsaver.com/profile/53608), the owner of [BeatSaver](https://beatsaver.com/), you can navigate to https://bookmarks.topc.at/ where you can access your legacy BeastSaber bookmarks! This page includes a basic initial listing of accounts and a search bar at the top.
+
+<p align="center">
+    <a href="https://bookmarks.topc.at/"><img src="/uploads/bookmarks1.png"></a>
+</p>
+
+Using the search bar at the top, type in your username. This will filter out the other accounts and display yours, along with a count of how many maps were there.
+
+<p align="center">
+    <img src="/uploads/bookmarks2.png">
+</p>
+
+To download your list, just click on the name. In your Downloads, you'll get a `.bplist` file which contains the hashes (or the unique identifier) for each of the maps that were in your bookmarks.
```

Comment:
For me, when I search something, it can take quite some time for it to appear without any loading indicator. I wouldn't be surprised if that would lead quite a few people dropping in and asking why it doesn't work. Maybe putting a hint here that it might be loading a bit without any visual indication could eliminate that?